python/appledoublekaitai/structs/apple_single_double.py

```diff
@@ -0,0 +1,136 @@
+# This is a generated file! Please edit source .ksy file and use kaitai-struct-compiler to rebuild
```

Comment:
IMHO the ksy file should be checked in, and this file should be generated as part of the build process.

Reply:
Checking in the ksy file has a lower priority in this epic and will be done as a different deliverable (see HANSKEN-20926 in Jira) and as such it will be done in a different PR. The ksy file won't be checked in in this pr.

Reply:
HANSKEN-20926, processing the ksy file as part of the build process, is planned to be done this week.